weatherman.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+from code_files.weather_parser import WeatherParser
+from code_files.weather_reporter import WeatherReporter
+from code_files.report_calculator import ReportCalculator
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("extract_to", type=str)
```

Comment:
Let's also add help messages for users.

Reply:
[Added help messages](https://github.com/ttqureshi/WeatherMan/pull/1/commits/90150ed551723c3fabc39a9b62a6234e3180bc01)